pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+|                                     | Configuration                              |                           |
+|                                     | DissolvingProcess                          |                           |
+|                                     | InformationObject                          |                           |
+|                                     | Instrument                                 |                           |
+|                                     | MassSpectrometry                           |                           |
+|                                     | MassSpectrometryConfiguration              |                           |
+|                                     | MaterialProcessing                         |                           |
+|                                     | MobilePhaseSegment                         |                           |
+|                                     | NucleotideSequencing                       |                           |
+|                                     | PortionOfSubstance                         |                           |
+|                                     | ProtocolExecution                          |                           |
+|                                     | StorageProcess                             |                           |
+|                                     | WorkflowExecution                          |                           |
+
+For example, `MetagenomeAnnotationActivity` used to be located in the hierarchy
+`Activity.WorkFlowExecutionActivity`, but now it and 11 similar classes are located in
+`NamedThing.PlannedProcess.WorkflowExecution`. The word `Activity`, inherited from the Provenance Ontology, has largely
+been removed from the schema, as `PlannedProcess` from the Ontology of Biomedical Investigations now plays a more
+important organizational role.
+
+## v10.9.1
+
+![Activity.png](Activity.png)
+
+`ProtocolExecution` has been added as a `PlannedProcess` for aggregating other processes together when they follow the
+same `Protocol`.
+
+![ProtocolExecution.png](ProtocolExecution.png)
+
+Several other kinds of processual classes were also migrated into the `NamedThing.PlannedProcess`, with or without
+renaming or other refactoring. For example, the fairly vague root class `OmicsProcessing` is now
+`NamedThing.PlannedProcess.DataGeneration`, which now has the following subclasses:
+
+- `MassSpectrometry`
+- `NucleotideSequencing`
+
+The leagcy `BiosampleProcessing` class (including `Pooling`, `LibraryPreparation`) and some classes that were previously
+direct subclasses of `PlannedProcess` (`Extraction`, `SubSamplingProcess`, `MixingProcess`, `FiltrationProcess`,
+`ChromatographicSeparationProcess`) have been re-rooted into a new `MaterialProcessing` class.
+
+`DissolvingProcess` and `ChemicalConversionProcess` are new subclasses of `MaterialProcessing` as of v11.
```

Comment:
I'm wondering whether the "v11" is implicit (I think all these statements are about things that changed between the legacy schema and the Berkeley schema).